test/separate_compilation/serialization/testGenLib.both.good

```diff
@@ -1,98 +1,4 @@
-Parsed file: $CHPL_HOME/modules/dists/BlockDist.chpl
-Parsed file: $CHPL_HOME/modules/dists/DSIUtil.chpl
-Parsed file: $CHPL_HOME/modules/dists/SparseBlockDist.chpl
-Parsed file: $CHPL_HOME/modules/internal/ArrayViewRankChange.chpl
-Parsed file: $CHPL_HOME/modules/internal/ArrayViewReindex.chpl
-Parsed file: $CHPL_HOME/modules/internal/ArrayViewSlice.chpl
-Parsed file: $CHPL_HOME/modules/internal/Atomics.chpl
-Parsed file: $CHPL_HOME/modules/internal/AtomicsCommon.chpl
-Parsed file: $CHPL_HOME/modules/internal/ByteBufferHelpers.chpl
-Parsed file: $CHPL_HOME/modules/internal/Bytes.chpl
-Parsed file: $CHPL_HOME/modules/internal/BytesCasts.chpl
-Parsed file: $CHPL_HOME/modules/internal/BytesStringCommon.chpl
-Parsed file: $CHPL_HOME/modules/internal/CString.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelArray.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelAutoAggregation.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelAutoLocalAccess.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelBase.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelContext.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelDebugPrint.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelDistribution.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelDomain.chpl
-Parsed file: $CHPL_HOME/modules/internal/ChapelGpuSupport.chpl
```

Comment:
The Chapel standard library isn't being passed in as a .dyno file, so I would expect it to get parsed. This change is concerning. Is this because these modules are dead? If so, we need to use a function to ensure at least some standard library files are used, so that this test remains viable.. Else, are these files parsed, and not printed? If so, the printing needs to be fixed for this test to work.

Reply:
I prediffed out all of these to keep tracking them from being a maintenance burden. The prediff now greps for `.chpl` files of interest, and I only included `ChapelStandard` in that list from the bundled modules.

The current contents of this file in this PR are this:

```
Parsed file: $CHPL_HOME/modules/internal/ChapelStandard.chpl
Parsed file: testGenLib.chpl
MYPREFIX: doMyM-doMyM-doMyM
MYPREFIX: doMrehtO-doMrehtO-doMrehtO
```

The `ChapelStandard` line indicates that the standard library is being parsed.